src/hotspot/share/prims/jvmtiThreadState.cpp

```diff
@@ -1041,6 +1041,14 @@ oop JvmtiThreadState::get_thread_oop() {
   return _thread_oop_h.resolve();
 }
 
+void JvmtiThreadState::update_thread_oop_during_vm_start(JavaThread* thread) {
+  assert(thread->threadObj() != nullptr, "santity check");
+  if (thread->jvmti_thread_state()->get_thread_oop() == nullptr) {
```

Comment:
Nit: Replace `thread->jvmti_thread_state()->get_thread_oop()` with `get_thread_oop()`.

Reply:
I realised that thread is already in this state, so no need to set it as a parameter.